pages/support/ticket-sidebar.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div>
+    <div
+      v-if="fields.orderId"
+      class="u-mb"
+    >
+      <h2 class="u-fs-lg u-mb-sm">Order {{ fields.orderId }}</h2>
+
+      <div>
+        <a
+          :href="`https://admin.system76.com/fulfillment/orders/${fields.orderId}`"
+          class="c-btn"
+          target="_blank"
+        >
+          LCARS
+        </a>
+
+        <a
+          :href="`https://system76.com/admin/service/order/${fields.orderId}`"
+          class="c-btn"
+          target="_blank"
+        >
+          Joshua
+        </a>
+      </div>
+    </div>
+
+    <div
+      v-if="user.id"
+      class="u-mb"
+    >
+      <h2 class="u-fs-lg u-mb-sm">User {{ user.id }}</h2>
+
+      <div>
+        <a
+          :href="`https://admin.system76.com/account/customers/${user.id}`"
```

Comment:
We should probably make the host configurable. We went to a lot of trouble to not configure field names but now there is a required code change per environment.

We should refactor this stuff to take advantage of configuration and simpler code.

Reply:
I second this. I'll update / simplify this in another PR